metricflow-semantics/metricflow_semantics/query/validation_rules/query_validator.py

```diff
@@ -65,7 +65,18 @@ def _default_handler(self, node: GroupByItemResolutionNode) -> MetricFlowQueryRe
 
     @override
     def visit_measure_node(self, node: MeasureGroupByItemSourceNode) -> MetricFlowQueryResolutionIssueSet:
-        return self._default_handler(node)
+        with self._path_from_start_node_tracker.track_node_visit(node) as current_traversal_path:
```

Comment:
Just curious: what is this `track_node_visit` for? Just for our own debugging?

Reply:
It's both for debugging and providing context to validation error messages.